Source/WTF/wtf/text/StringImpl.h

```diff
@@ -1152,6 +1156,36 @@ inline void StringImpl::copyCharacters(DestinationCharacterType* destination, co
         }
         memcpy(destination, source, numCharacters * sizeof(DestinationCharacterType));
     } else {
+#if CPU(ARM64)
+        // SIMD Upconvert.
+        if constexpr (std::is_same_v<SourceCharacterType, LChar> && std::is_same_v<DestinationCharacterType, UChar>) {
+            const auto* const end = destination + numCharacters;
```

Comment:
Not sure the second `const` makes this easier to read.

Reply:
Dropped.